app/src/main/res/values/strings.xml

```diff
@@ -388,14 +388,14 @@
     <string name="settings_file_replacement_character_title">Replacement character</string>
     <string name="charset_letters_and_digits">Letters and digits</string>
     <string name="charset_most_special_characters">Most special characters</string>
-    <string name="toast_no_player">No app installed to play this file</string>
+    <string name="toast_no_player">No apps could play this file</string>
     <!-- About -->
     <string name="title_activity_about">About NewPipe</string>
     <string name="action_settings">Settings</string>
     <string name="action_about">About</string>
     <string name="title_licenses">Third-party Licenses</string>
     <string name="copyright">© %1$s by %2$s under %3$s</string>
-    <string name="error_unable_to_load_license">Could not load license</string>
+    <string name="error_unable_to_load_license">Could not load the GPLv3+ license</string>
```

Comment:
What? This string is used to signal errors for other licenses, too, i.e. those for the used libraries

Reply:
```suggestion
    <string name="error_unable_to_load_license">One or more licenses could not be loaded</string>
```